BLE_GattServer/main.cpp

```diff
@@ -0,0 +1,668 @@
+#include <stdio.h>
+
+#include "platform/Callback.h"
+#include "events/Eventqueue.h"
+#include "platform/NonCopyable.h"
+
+#include "ble/BLE.h"
+#include "ble/Gap.h"
+#include "ble/GattClient.h"
+#include "ble/GapAdvertisingParams.h"
+#include "ble/GapAdvertisingData.h"
+#include "ble/GattServer.h"
+
+using mbed::callback;
+
+/**
+ * A Clock service that demonstrate the GattServer features.
+ *
+ * The clock service host three characteristics that respectively model the
```

Comment:
language nitpick: the word respectively would only work if you listed the three characteristics in question (and would have to come after the word "model")

Reply:
Cheers, I will amend the comment.